assets/js/modules/adsense/datastore/base.test.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * `modules/adsense` base data store tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	provideSiteInfo,
+} from '../../../../../tests/js/utils';
+import { MODULES_ADSENSE } from './constants';
+
+describe( 'modules/adsense base data store', () => {
+	let registry;
+	let store;
+
+	beforeEach( () => {
+		jest.resetModules();
+
+		registry = createTestRegistry();
+		provideSiteInfo( registry );
+	} );
+
+	it( 'does not define the admin page when unified dashboard is enabled', () => {
+		const { enabledFeatures } = require( '../../../features' );
+		enabledFeatures.add( 'unifiedDashboard' );
+
+		store = require( './base' ).default;
+		registry.registerStore( MODULES_ADSENSE, store );
+
+		expect( store.selectors.getAdminScreenURL() ).toBe(
+			`http://example.com/wp-admin/admin.php?page=googlesitekit-dashboard`
```

Comment:
Please, use single quotes for bare strings: 
```suggestion
			'http://example.com/wp-admin/admin.php?page=googlesitekit-dashboard'
```

Reply:
Thanks for pointing this out. Seems like something we should enforce with a linting rule though, what do you think?

Reply:
That's done too :+1:

Reply:
@techanvil, yes, this should be configured in eslint rules. Could you please create a new ticket to update eslint rules?

Reply:
@eugene-manuilov thanks - I've raised an issue for it: https://github.com/google/site-kit-wp/issues/4350